pkg/image/apiserver/apiserver.go

```diff
@@ -165,10 +175,13 @@ func (c *completedConfig) newV1RESTStorage() (map[string]rest.Storage, error) {
 		whitelister = whitelist.WhitelistAllRegistries()
 	}
 
+	imageLayerIndex := imagestreametcd.NewImageLayerIndex(imageV1Client.Image().Images())
+	c.ExtraConfig.startFns = append(c.ExtraConfig.startFns, imageLayerIndex.Run)
```

Comment:
This function is only called once (the public access protects it), so you can add the poststarthook right here.

Reply:
I don't have access to start it here.